tests/quality_pragmaitc/test_refinement.py

```diff
@@ -88,23 +89,16 @@ def test_shouldAddSeveralContextsAtOnce():
     context2 = Context("C2")
 
     task = Task()
-    originalSize = 0
-
-    if None is task.getApplicableContext():
-        originalSize = len(task.getApplicableContext())
 
-    set = []
+    assert task.applicableContext is None
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/gh/lesunb/p2psimpy/pullRequest?prid=4895731)